rust/test/video/inc/apng_test.rs

```diff
@@ -67,4 +154,6 @@ tests_index!
 {
   basic_rgb,
   basic_rgba,
+  basic_imagesize,
```

Comment:
You need to extend the existed tests. 
The written tests duplicate existed tests, Please, remove extra test code.

Reply:
Ok!